app-arch/7zip/7zip-23.01.ebuild

```diff
@@ -17,7 +17,7 @@ S="${WORKDIR}"
 
 LICENSE="LGPL-2 BSD rar? ( unRAR )"
 SLOT="0"
-KEYWORDS="~amd64"
+KEYWORDS="~amd64 ~arm64"
```

Comment:
I don't think this is enough because there are x64 specific assembly that gets used when `uasm` or `jwasm` is enabled.

Reply:
Ah, didn't notice that :crying_cat_face:

Is the best path forward here to 'mask' those use flags on arm64? Or is that unreasonable? It does work without those for me, but not sure if there's something I'm missing.

Reply:
Haven't looked at these flags myself but if amd64-only then I'd mask in profile/arch/base and unmask in arch/amd64 so it won't come back as an issues for every new keywords.

Odds are we'll want more keywords at some point given feels like it should replace the dead p7zip in unpacker.eclass eventually (which currently has all keywords except mips and m68k for that purpose).

Reply:
> It does work without those for me, but not sure if there's something I'm missing.

The asm bits are optional and for speed. It's supposed to still work without them.

> Is the best path forward here to 'mask' those use flags on arm64? Or is that unreasonable? 

I think masking is the recommended way to deal with such issues: https://devmanual.gentoo.org/general-concepts/use-flags/index.html#noblah-use-flags

I'd go with what @ionenwks suggested.

Reply:
Latest commit has the suggested way of masking the USE flags! Thanks for the pointers all :)